app/theme/client/imports/general/base.css

```diff
@@ -239,7 +239,7 @@ button {
 	.rc-old .main-content-flex {
 		flex: 1 0 auto !important; /* 1 */
 
-		height: auto !important; /* 1 */ 
+		height: auto !important; /* 1 */
```

Comment:
Didn't this this change, doesn't seem relevant to this PR. If its by mistake, please remove it